docs/DeveloperGuide.md

```diff
@@ -154,6 +161,76 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Schedule feature
+
+Users can view a calendar that contains all existing lessons to visualise their schedule and plan ahead.
+
+#### Current Implementation
+
+TAB uses the [CalendarFX](https://dlsc.com/products/calendarfx/) library to implement its schedule view.
+Each lesson stored in the `Model` component is also converted to a CalendarFX `Entry` and maintained in the `CalendarEntryList` of `AddressBook`. 
+The `Ui` component shows this list of entries in the `SchedulePanel` using CalendarFX's `DetailedWeekView`.
```

Comment:
Can explain in more details like how `CalendarEntryList` was derived from lessons etc.